packages/mongo-nav/src/MongoNav.story.tsx

```diff
@@ -21,6 +21,11 @@ storiesOf('MongoNav', module).add('Default', () => (
           Object.values(Product),
           Product.Cloud,
         )}
+        activePlatform={select(
```

Comment:
Super silly nit, can we put this above `activeProduct` so the circle of ownership goes from top down, where the platform contains the product, contains the nav.

Reply:
👍